zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryResponseMapper.java

```diff
@@ -334,6 +337,54 @@ private static DecisionInstanceItem toDecisionInstance(final DecisionInstanceEnt
             (entity.decisionType() == null)
                 ? null
                 : DecisionInstanceTypeEnum.fromValue(entity.decisionType().name()))
-        .result(entity.result());
+        .result(entity.result())
+        .evaluatedInputs(toEvaluatedInputs(entity.evaluatedInputs()))
+        .matchedRules(toMatchedRules(entity.evaluatedOutputs()));
   }
+
+  private static List<EvaluatedDecisionInputItem> toEvaluatedInputs(
+      final List<DecisionInstanceInputEntity> decisionInstanceInputEntities) {
+    if (decisionInstanceInputEntities == null) {
+      return null;
+    }
+    return decisionInstanceInputEntities.stream()
+        .map(
+            input ->
+                new EvaluatedDecisionInputItem()
+                    .inputId(input.id())
+                    .inputName(input.name())
+                    .inputValue(input.value()))
+        .toList();
+  }
+
+  private static List<MatchedDecisionRuleItem> toMatchedRules(
```

Comment:
@ralfpuchert the `MatchedDecisionRules` are flattened by Operate importer to a List of EvaluatedOutputs, here I try to regroup them by ruleId/ruleIndex, can you confirm this logic is correct?